databases/week3/dbweek3.sql

```diff
@@ -0,0 +1,356 @@
+SHOW DATABASES
+
+USE MealSharing
+
+CREATE TABLE meal (
+id INT(10) PRIMARY KEY AUTO_INCREMENT,
+title VARCHAR(255) NOT NULL,
+description TEXT null,
+location VARCHAR(255)NOT NULL,
+whenn DATETIME,
+maxreservations INT(10),
+price DECIMAL(5 , 2),
```

Comment:
it will work fine? (1023 is < 5 , 31 = 2) no? I can also do a (10, 2) but then we are talking a 2 Michelin star at least in my opinion.

Reply:
The thing is that the 5 is the total number of digits in the price, and since you specify that the price will have 2 decimal places, the left side of the price can only have 3 digits, so if you try adding a price of 1023,31, it will fail ;p 
DECIMAL(6,2) is the minimum i'd say, but to be sure i'd set it to DECIMAL(7,2). A meal of 10000,23 dkk is indeed expensive, but at least nothing should crash when you insert prices xD